pom.xml

```diff
@@ -411,6 +421,12 @@
             <activation>
                 <jdk>1.8</jdk>
             </activation>
+
+            <!-- Exclude the example app from javadoc packages -->
+            <modules>
+                <module>graphql-kotlin-schema-generator</module>
```

Comment:
and this should be the same as -> https://github.com/ExpediaDotCom/graphql-kotlin/blob/master/graphql-kotlin-spring-example/pom.xml#L89